WeakAuras/Transmission.lua

```diff
@@ -145,7 +145,7 @@ local function filterFunc(_, event, msg, player, l, cs, t, flag, channelId, ...)
   local remaining = msg;
   local done;
   repeat
-    local start, finish, characterName, displayName = remaining:find("%[WeakAuras: ([^%s]+) %- ([^%]]+)%]");
+    local start, finish, characterName, displayName = remaining:find("%[WeakAuras: ([^%s]+) %- (.*)%]");
```

Comment:
This fails on the following chat message:

> Hey guys, look at these great auras: [WeakAuras: EmptyRivers - Aura 1] [WeakAuras: EmptyRivers - Aura 2]

The first link will request an aura named `Aura 1] [WeakAuras: EmptyRivers - Aura 2]`. I'm not entirely sure how the second link will behave, though.

Reply:
i have always used weakauras links with nothing else than 1 link, i didn't know 2 links in same line was supported tbh

Reply:
It also fails if the chat message has a ] in it after the link. So "Guys, install this: [WeakAuras: EmptyRivers - Aura 1]  [].

The "proper" way to do this is to escape any ] in the name of the aura and to properly parse the msg.